local-cli/runIOS/runIOS.js

```diff
@@ -160,30 +171,43 @@ function buildProject(xcodeProject, udid, scheme, configuration = 'Debug', launc
       '-derivedDataPath', 'build',
     ];
     console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
+    if (!verbose) {
+      var xcpretty = xcprettyAvailable();
+      if (xcpretty) {
+        var xcprettyProcess = child_process.spawn('xcpretty', [], { stdio: ['pipe', process.stdout, process.stderr] });
```

Comment:
Normally I agree, but I was hoping to avoid running the synchronous `xcprettyAvailable` function if I could avoid it when verbose is enabled. I can flatten this out if this is not considered an issue.